testing/features/automated_release_process.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Automated Release Process
+
+This feature is to automate the design system release process, removing repeatable manual developer actions. 
+
```

Comment:
Adding here as a comment in case we need to add the original requirements on Jira ticket from Dan in some form:
- As a Design System Developer I can release the Design System using an automated process so that the release artifacts and process are consistent.
- As a Design System Developer I can release the Design System using an automated process so I can save time and do something else whilst the release is happening.

Reply:
Fine to add these I think - you can combine them into one tho?

Reply:
Yep - can combine into 1. Only reservation was about specifying the solution here but I suppose it's the core of the feature!

Reply:
added below